src/titan_checkpoint_impl.cc

```diff
@@ -83,20 +85,29 @@ Status TitanCheckpointImpl::CreateTitanManifest(
 }
 
 // Builds an openable checkpoint of TitanDB
-Status TitanCheckpointImpl::CreateCheckpoint(const std::string& checkpoint_dir,
-                                             uint64_t log_size_for_flush) {
-  DBOptions db_options = db_->GetDBOptions();
+Status TitanCheckpointImpl::CreateCheckpoint(
+              const std::string& base_checkpoint_dir,
+              const std::string& titan_checkpoint_dir,
```

Comment:
Can we get the titan relative dir by the `dirname_` in `TitanDBImpl`? So we can construct the titan checkpoint dir instead of specifying one.

Reply:
I've thought about this before, but I think `dirname_` can be specified by the user to any absolute path, it's only specified as a relative location by default.